packages/protocol/contracts/governance/Governance.sol

```diff
@@ -59,8 +67,8 @@ contract Governance is IGovernance, Ownable, Initializable, UsingBondedDeposits,
   // TODO(asa): Reduce storage usage here.
   struct VoteTotals {
     uint256 abstain;
-    uint256 yes;
     uint256 no;
+    uint256 yes;
```

Comment:
I had attempted to save bytecode in `vote()` by indexing into the struct using the `VoteValue` enum, but it didn't work well. Will revert this change because it's pretty unnecessary.